health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserManagement.js

```
@@ -0,0 +1,60 @@
+import { useMyContext } from "../../utils/context";
+
+const UserManagement = () => {
+    const { t } = useTranslation();
+    const location = useLocation()
+    const moduleName = Digit?.Utils?.getConfigModuleName() || "commonSanitationUiConfig"
+    const tenant = Digit.ULBService.getStateId();
+    
+
+    
+    const config = UserManagementConfig?.UserManagementConfig?.[0];
+    debugger
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)